### DISCUSSION TITLE
POD not installed in my Ionic App

### DISCUSSION THREAD

**Discussion by achakra21** [ionic-team/capacitor#4734] [General]
This has been written in my plugin.xml but its not. adding anything in pod file or neither on POD folder the sdk

```
<platform name="ios">
        <config-file target="config.xml" parent="/*">
            <feature name="FlurryAnalyticsPlugin">
                <param name="ios-package" value="FlurryAnalyticsPlugin"/>
            </feature>
        </config-file>
        <!--<header-file src="src/ios/Flurry.h" />-->
        <header-file src="src/ios/FlurryAnalyticsPlugin.h"/>
        <source-file src="src/ios/FlurryAnalyticsPlugin.m"/>
        <pod id="Flurry-iOS-SDK/FlurrySDK"/>
    </platform>
```

**Comment by jcesarmobile** [comment]
That's not valid plugin.xml syntax, check plugin.xml docs for podspec, pod should be inside a podspec

https://cordova.apache.org/docs/en/10.x/plugin_ref/spec.html#podspec-ios

**Reply by achakra21** [reply]
I have followed the with capacitor project it seems not working here is below my pod file looks like
```
<platform name="ios">
        <config-file target="config.xml" parent="/*">
            <feature name="FlurryAnalyticsPlugin">
                <param name="ios-package" value="FlurryAnalyticsPlugin"/>
            </feature>
        </config-file>
        <!--<header-file src="src/ios/Flurry.h" />-->
        <header-file src="src/ios/FlurryAnalyticsPlugin.h"/>
        <source-file src="src/ios/FlurryAnalyticsPlugin.m"/>
        
<podspec>
     
      <pods use-frameworks="true">
      <pod name="Flurry-iOS-SDK/FlurrySDK"/>

      </pods>
    </podspec>
```

**Reply by jcesarmobile** [reply]
I've tried with that and I see FlurrySDK listed in my Podfile.lock (which means it was installed).
So, what do you mean by "not working"?

**Reply by achakra21** [reply]
If you use https://github.com/blakgeek/cordova-plugin-flurryanalytics in your ionic project the pod won’t install it . Even after changing the pod I’d to name Use it any demo ionic capacitor project .you see it when you run it as iOS project .

**Reply by jcesarmobile** [reply]
Sorry, I don’t understand what you mean. That plugin has the pod as in your first message, which I told you wasn’t valid.

**Reply by achakra21** [reply]
This is plugin i was using try to use it any project with modification of pod id to pod name, after that it install sdk for you will find an error flurry.h not found but which is already there in sdk.
